test/pylib/scylla_cluster.py

```diff
@@ -650,11 +650,11 @@ async def stop_gracefully(self) -> None:
     def _seeds(self) -> List[str]:
         return [server.ip_addr for server in self.running.values()]
 
-    async def add_server(self, replace_cfg: Optional[ReplaceConfig] = None, cmdline: Optional[List[str]] = None) -> ServerInfo:
+    async def add_server(self, replace_cfg: Optional[ReplaceConfig] = None, cmdline: Optional[List[str]] = None, config: dict[str, str] = None) -> ServerInfo:
```

Comment:
As above, `None` is not a valid value for `dict[str, str]`

Reply:
fixed